index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Luis Laverde's Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+  <nav class="navbar">
+    <div><img src="./imgs/my-logo/my-logo.png" /></div>
+    <div class="hamburguer-menu"><img src="./imgs/hamburguer-menu/hamburguer-menu-icon-enabled.png" /></div>
+  </nav>
```

Comment:
- Great job warping the hamburger with `nav` 👍 I think it would be better if you wrap the `nav` with `header` for semantic reasons  [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.
- please also move the logo outside the `nav` as it is not part of the navigation and wrap it with an `a` tag for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
### Check the bellow code snippet 👇
```
<body>
    <header>
        <a>logo</a>
         <nav>
               hamburger
          </nav>
     </header>
     <main>
          <section>
              .....
           </section>
      </main>
</body>

Reply:
It's fixed